src/_articles/language/mixins.md

```diff
@@ -13,20 +13,17 @@ Looking for an introduction to mixins?
 See the Language Tour section on [Adding features to a class: mixins][].
 
 This article provides a brief theoretical presentation of the evolution of mixins in Dart.
-
-Feature evolution notes:
-
-- Support for `--supermixin` is being rescinded in Dart 2.
-- Dart 1.13 and greater (< Dart 2) supports mixins that can extend from classes
-other than `Object`, and can call `super.method()`. This support is only
-available by default in the Dart VM and in Analyzer behind a flag.
-More specifically, it is behind the `--supermixin` flag in the
-command-line analyzer. It is also available in the analysis server,
-behind a client-configurable option. For example, in Atom use the
-analysis.updateOptions request to set the enableSuperMixins option to true.
-Dart2js and dartdevc (also known as _DDC_) do not support this yet.
-- Dart 1.12 or lower supports mixins that must extend Object,
-and must not call super().
+Support for mixins changed in Dart 1.13, and again in Dart 2:
+
+- Dart 1.12 or lower supports mixins that must extend `Object`, and must not
+  call `super()`.
+- Dart 1.13 or greater (< Dart 2) supports mixins that can extend from classes
+  other than `Object`, and can call `super.method()`. This support is only
+  available by default in the Dart VM and in Analyzer behind a flag. More
+  specifically, it is behind the `--supermixin` flag in the command-line
+  analyzer. It is also available in the analysis server, behind a
+  client-configurable option. Dart2js and dartdevc do not support super mixins.
+- Dart 2 deprecates `--supermixin`.
```

Comment:
Is it really deprecated, or is it not not allowed?

Reply:
The option is still available:

```console
> dart --version
Dart VM version: 2.0.0-dev.66.0 (Fri Jun 29 11:19:05 2018 +0200) on "macos_x64"
> dartanalyzer -h -v 2>&1 | grep mixin
    --supermixin                               Relax restrictions on mixins (DEP 34).
```

But, as @eernstg [put it](https://github.com/dart-lang/site-www/issues/946#issuecomment-398805176):

> `--supermixins`... is going away.

So, my understanding is that this feature will be deprecated in Dart 2. Maybe you'd prefer "Dart 2 plans to deprecate ..."?

Reply:
"Dart 2" doesn't plan. :) Let's leave it as you had it.

Reply:
Drive-by comment, just noting that you were discussing the status of `--supermixin`: The language specification has the following (line 2610 in commit 201fb2833c7e8956997e243bcb1a135cff98e5a1):
```
It is a compile-time error to derive a mixin from a class which has a
superclass other than \code{Object}.
```
which shows that `--supermixin` is not part of Dart 2. It is a special extension which is used by Flutter, and all its usages will need to be adjusted to use the upcoming `mixin` declaration. Because of that, we should not give anyone the impression that `--supermixin` is an upcoming feature which will be part of a future Dart and will hence be enabled by default, they should at most assume that `--supermixin` will give them access to features that they will get in a different syntactic form in the future (it is expected to be part of the first significant language update after Dart 2).

Reply:
A better reference is the feature specification: https://github.com/dart-lang/sdk/blob/master/docs/language/informal/mixin-declaration.md. That's very nearly what we will actually do (the 'very nearly' part is just that this document may still be updated on a few details).